app/styles/tailwind.css

```
@@ -510,277 +510,17 @@ video {
-  display: none;
-}
-
-/* Entry
-------------------------------------------------------------------------------*/
-
-@supports selector(:has(*)) {
-  .theme-select {
-    display: block;
-  }
-}
+@import 'components/button';
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)